Historical.md

```diff
@@ -178,7 +178,7 @@ This is a sibling list of [awesome-AutoHotkey](https://github.com/ahkscript/awes
 
 ### Documentation
 
-* [http://autohotkey.free.fr/docs/](http://autohotkey.free.fr/docs/) - Documentation for AHK 1.0 in French
+* [https://autohotkey.free.fr/docs/](https://autohotkey.free.fr/docs/) - Documentation for AHK 1.0 in French
```

Comment:
Everything looks great! Sadly `http://autohotkey.free.fr/docs/` must remain normal `http / port 80` protocol since it does not support SSL.